src/options/smt_options.toml

```diff
@@ -9,6 +9,66 @@ name   = "SMT Layer"
   default    = "false"
   help       = "eliminate functions by ackermannization"
 
+[[option]]
+  name       = "computePartitions"
+  category   = "expert"
+  long       = "compute-partitions=N"
+  type       = "uint64_t"
+  default    = "0"
+  help       = "make n partitions"
+
+[[option]]
+  name       = "numChecks"
+  category   = "expert"
+  long       = "num-checks=N"
+  type       = "uint64_t"
+  default    = "1"
+  help       = "number of standard effort checks until partitioning"
```

Comment:
Can we give that a better name? It seems to me that `numChecks` is very vague/generic... maybe `checksBeforePartition`?
Also, whether this are standard effort checks seems to depend on `partitionCheck`